src/main/java/bookstore/services/categoryservice/CategoryService.java

```diff
@@ -0,0 +1,68 @@
+package bookstore.services.categoryservice;
```

Comment:
```suggestion
package bookstore.services.category;
```
the same for other packages ;)